cmd/apiserver/main.go

```diff
@@ -130,6 +130,12 @@ func realMain(ctx context.Context) error {
 	}
 	rateLimit := httplimiter.Handle
 
+	// Install HSTS headers in production
+	if !config.DevMode {
```

Comment:
guessing this won't matter as non-browsers will ignore...

Reply:
but also no harm

Reply:
You don't want to set it in dev because localhost w/ hsts doesn't work.

Reply:
I think Mike was saying that no one will call the API in the browser so it probably doesn't matter